contributor_docs/zh/steward_guidelines.md

```diff
@@ -174,62 +180,66 @@ grunt.registerTask('build', [
+- `uglify`接收`browserify:min`的输出文件，并将其缩小为最终的 p5.min.js（此步骤的配置在主 Gruntfile.js 中）。
+- `browserify:test`构建的版本与完整的 p5.js 版本相同，只是添加了用于测试代码覆盖率报告的代码（使用 [Istanbul](https://istanbul.js.org/)）。
 
-在browserify步骤中，除了将各种文件合并为一个文件外，还执行了其他几个步骤。首先，使用`brfs-babel`将`fs.readFileSync()`节点.js特定代码的使用替换为文件的实际内容。这主要用于WebGL代码，以内联着色器代码，同时将它们保留为单独的文件。
+首先，使用`brfs-babel`将`fs.readFileSync()`节点.js特定代码的使用替换为文件的实际内容。这主要用于WebGL代码，以内联着色器代码，同时将它们保留为单独的文件。
 
-接下来，使用Babel将来自node_modules的所有依赖项的源代码进行转译，以匹配package.json中定义的Browserslist要求，并将ES6导入语句转换为browserify理解的CommonJS `require()`。这还使得可以使用ES6及更高版本中的较新语法，而不必过多考虑浏览器兼容性。
+接下来，使用 Babel 将来自 node_modules 的所有依赖项的源代码进行转译，以匹配 package.json 中定义的 [Browserslist](https://browsersl.ist/) 要求，并将 ES6 导入语句转换为 browserify 理解的 CommonJS `require()`。这还使得可以使用ES6及更高版本中的较新语法，而不必过多考虑浏览器兼容性。
 
-在捆绑之后但将捆绑代码写入文件之前，如果不打算缩小代码，则通过`pretty-fast`进行清理，以使最终格式更加一致（我们预计可以阅读和检查p5.js源代码，如果需要的话）。
+在捆绑之后但将捆绑代码写入文件之前，如果不打算缩小代码，则通过`pretty-fast`进行清理，以使最终格式更加一致（我们预计可以阅读和检查 p5.js 源代码，如果需要的话）。
 
-这里省略了一些小的详细步骤，您可以查看上面链接的browserify构建定义文件，以更详细地了解所有内容。
+这里省略了一些小的详细步骤，您可以查看上面链接的 browserify 构建定义文件，以更详细地了解所有内容。
 
 ```
 connect:server
 ```
-此步骤启动一个本地服务器，托管测试文件和构建的源代码文件，以便可以在Chrome中运行自动化测试。
+此步骤启动一个本地服务器，托管测试文件和构建的源代码文件，以便可以在 Chrome 中运行自动化测试。
 
 ```
 mochaChrome
 ```
-此步骤在[./tasks/test/mocha-chrome.js](./tasks/test/mocha-chrome.js)中定义。它使用Puppeteer来启动一个无头版本的Chrome，可以进行远程控制，并运行与`./test`文件夹中的HTML文件相关联的测试，包括对未缩小和缩小版本的库进行单元测试以及测试所有参考示例。
+此步骤在[./tasks/test/mocha-chrome.js](./tasks/test/mocha-chrome.js)中定义。它使用 Puppeteer 来启动一个无头版本的 Chrome，可以进行远程控制，并运行与`./test`文件夹中的 HTML 文件相关联的测试，包括对未缩小和缩小版本的库进行单元测试以及测试所有参考示例。
 
 ```
 mochaTest
 ```
-与`mochaChrome`不同，此步骤在node.js中运行，而不是在Chrome中运行，并且仅测试库中的一小部分功能。p5.js中的大多数功能都需要浏览器环境，因此只有在新的测试确实不需要浏览器环境时，才应扩展此测试集合。
+与`mochaChrome`不同，此步骤在 node.js 中运行，而不是在 Chrome 中运行，并且仅测试库中的一小部分功能。p5.js 中的大多数功能都需要浏览器环境，因此只有在新的测试确实不需要浏览器环境时，才应扩展此测试集合。
 
 ```
 nyc:report
 ```
-最后，在完成所有构建和测试之后，此步骤将收集`mochaChrome`对库的完整版本进行的测试覆盖率报告，并将测试覆盖数据打印到控制台。p5.js的测试覆盖率主要用于监控和提供一些额外的数据点，我们的目标不是达到100%的测试覆盖率。
+最后，在完成所有构建和测试之后，此步骤将收集`mochaChrome`对库的完整版本进行的测试覆盖率报告，并将测试覆盖数据打印到控制台。p5.js 的测试覆盖率主要用于监控和提供一些额外的数据点，我们的目标不是达到100%的测试覆盖率。
 
-以上涵盖了Gruntfile.js配置中的默认任务！
+以上涵盖了 Gruntfile.js 配置中的默认任务！
 
 ## 杂项任务
 如果需要，可以直接使用`npx grunt [step]`运行所有步骤、子步骤和子子步骤，尽管对于某些步骤而言，如果依赖于此链中的较早步骤，可能没有太大意义进行操作。还有一些未在上面提到但在某些情况下可能有用的任务。
 
 ```
 grunt yui:dev
 ```
-此任务将运行上述描述的文档和库构建，然后启动一个Web服务器，提供与网站上[http://localhost:9001/docs/reference/](http://localhost:9001/docs/reference/)上的参考页面功能相似的版本。然后，它将监视源代码的更改，并重新构建文档和库。
+此任务将运行上述描述的文档和库构建，然后启动一个 Web 服务器，提供与网站上[http://localhost:9001/docs/reference/](http://localhost:9001/docs/reference/)上的参考页面功能相似的版本。然后，它将监视源代码的更改，并重新构建文档和库。
 
-当您在内联文档中工作时，这很有用，因为您无需每次更改时将构建文件从p5.js存储库移动到本地的p5.js-website存储库并重新构建网站，而是可以在浏览器中预览您的更改。这样，您也可以更加确信您所做的更改可能会正确显示在网站上。请注意，这仅适用于对内联文档的修改，对参考页面本身的更改，包括样式和布局，应在网站存储库上进行并进行测试。
+当您在内联文档中工作时，这很有用，因为您无需每次更改时将构建文件从 p5.js 存储库移动到本地的 p5.js-website 存储库并重新构建网站，而是可以在浏览器中预览您的更改。这样，您也可以更加确信您所做的更改可能会正确显示在网站上。请注意，这仅适用于对内联文档的修改，对参考页面本身的更改，包括样式和布局，应在网站存储库上进行并进行测试。
```

Comment:
原文：`grunt` `yui:dev` is useful when you are working on the reference in the inline documentation because you don't have to move built files from the p5.js repository to a local p5.js-website repository and rebuild the website each time you make a change, and you can just preview your changes with this slightly simplified version of the reference in your browser. This way, you can also be more confident that the changes you made are likely to show up correctly on the website. Note that this is only meant for modifications to the inline documentation; changes to the reference page itself, including styling and layout, should be made and tested on the website repository.

有部分没翻译出来，请修改